end-to-end-tests/specs/home.spec.js

```diff
@@ -116,7 +116,7 @@ describe('homepage', function() {
         );
     });
 
-    describe('select all/deselect all functionality in study selector',function(){
+    describe.skip('select all/deselect all functionality in study selector',function(){
```

Comment:
?

Reply:
I actually by mistake added master new commits into this branch should i remove them all and do the changes which are only done by me.